scpdt/plugin.py

```diff
@@ -1,14 +1,18 @@
 """
 A pytest plugin that provides enhanced doctesting for Pydata libraries
 """
+import os
+import shutil
 
 from _pytest import doctest
 from _pytest.doctest import DoctestModule, DoctestTextfile
 from _pytest.pathlib import import_path
 from _pytest.outcomes import skip
 
-from scpdt.impl import DTChecker, DTConfig, DTParser, DTFinder
+from scpdt.impl import DTChecker, DTParser, DTFinder
+from scpdt.tests.conftest import config
```

Comment:
Nit: if this is a DTConfig instance, can we name it `dt_config` or some such, to avoid mixing it up with `config` objects coming from pytest or sphinx etc